fabfile.py

```diff
@@ -396,7 +386,7 @@ def _kernel_deps():
         'xmlto',
         'libgtk2.0-dev',
 	'libunwind-setjmp0-dev',
-	'libunwind7-dev',
+#	'libunwind7-dev',
```

Comment:
Couldn't tell you.  Does it really matter as far as fabfile is concerned since it'll be going away soon?

The package was added in https://github.com/ceph/autobuild-ceph/commit/bbf4fa412473c104779c10594249c222c9bbac1f